services/ecr.py

```diff
@@ -1,29 +1,82 @@
 from models import RuleCheckResult
 import boto3
+import botocore
 
 
-# client = boto3.client("")
+client = boto3.client("ecr")
 
 
 def ecr_private_image_scanning_enabled():
+    repositories = client.describe_repositories()
+    compliant_resource = []
+    non_compliant_resources = []
+
+    for repository in repositories["repositories"]:
+        if repository["imageScanningConfiguration"]["scanOnPush"] == True:
+            compliant_resource.append(repository["repositoryArn"])
+        else:
+            non_compliant_resources.append(repository["repositoryArn"])
+
     return RuleCheckResult(
-        passed=False, compliant_resources=[], non_compliant_resources=[]
+        passed=not non_compliant_resources,
+        compliant_resources=compliant_resource,
+        non_compliant_resources=non_compliant_resources,
     )
 
 
 def ecr_private_lifecycle_policy_configured():
+    repositories = client.describe_repositories()
+    compliant_resource = []
+    non_compliant_resources = []
+
+    for repository in repositories["repositories"]:
+        try:
+            response = client.get_lifecycle_policy(
+                registryId=repository["registryId"],
+                repositoryName=repository["repositoryName"],
+            )
+            compliant_resource.append(repository["repositoryArn"])
+        except botocore.errorfactory.LifecyclePolicyNotFoundException:
+            non_compliant_resources.append(repository["repositoryArn"])
```

Comment:
`botocore.errorfactory.LifecyclePolicyNotFoundException`에 접근 시 AttributeError가 발생합니다. 다음과 같이 수정해 볼 수 있어 보입니다.

```suggestion
        except Exception as e:
            if e.__class__.__name__ == "LifecyclePolicyNotFoundException":
                non_compliant_resources.append(repository["repositoryArn"])
            else:
                raise e
```